docs/internships/outreachy/getting-started.md

```diff
@@ -0,0 +1,46 @@
+---
+id: getting-starteds
+title: Our repositories
+sidebar_label: Getting Started
+---
+
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
+1. [Talawa API github](https://github.com/PalisadoesFoundation/talawa-api)
+1. [Talawa Admin github](https://github.com/PalisadoesFoundation/talawa-admin)
+1. [Talawa Docs github](https://github.com/PalisadoesFoundation/talawa-docs).
+## Getting started with our repositories
+
+In our github repositories we have a few files that include guidelines and rules to guide applicants on how to contribute and the rules to follow in other to open a acceptable pull request and give a meaningful contribution.
+
+1. **CODE_OF_CONDUCT.md**
+
+This file contains guidelines of how each contributor in the community should behave and communicate with each other. It also states the consequences of going against these rules. By doing this we aim to provide a harassment-free environment and also an inclusive, healthy and diverse community.
+
+2. **CONTRIBUTING.md**
+
+Everything you need to know to start contributing to our projects can be found in this file.
+Some of them include the contribution code and ways to communicate with the community.
+
+3. **DOCUMENTATION.md**
+
+This file points out some useful tips on where to find our documentation and how to use Docusaurus.
+
+4. **INSTALLATION.md**
+
+Installation guide for the projects can be found in this file. Here, the dependencies to installand the versions, essential configurations are outlined step by step for different operating systems inorder for applicants to successfully run the projects locally on their machines.
+
+5. **ISSUE_GUIDELINES.md**
+
+The steps to create a new issue and how you can work with issues are outlines in this file. For instance, applicants should only work on issues assigned to them.
+
+6. **PR_GUIDELINES.md**
+
+Pull request guidelines are stated here and should be followed to the letter. The steps to open an acceptable pull request are also outlined here.
+
+7. **README.md**
```

Comment:
1. Please put this as the first point
2. Mention as a bullet that people can join our slack channel by following the link on the README.md file of the Talawa repo